hppc/src/main/java/com/carrotsearch/hppc/IntGrowableArray.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * HPPC
+ *
+ * Copyright (C) 2010-2022 Carrot Search s.c.
+ * All rights reserved.
+ *
+ * Refer to the full license file "LICENSE.txt":
+ * https://github.com/carrotsearch/hppc/blob/master/LICENSE.txt
+ */
+package com.carrotsearch.hppc;
+
+import java.util.Arrays;
+
+/**
+ * Basic growable int array helper for HPPC templates (so before {@code IntArrayList} is generated).
+ */
+public class IntGrowableArray implements Accountable {
```

Comment:
Used before IntArrayList is template-generated.

Reply:
I may take a look if this can be improved somehow. Could definitely be a separate module for pgm only... but perhaps we can make it work with more than one source folder and cascaded generation as well.